packages/algorand-js/src/errors/errors-list.ts

```diff
@@ -68,10 +68,9 @@ minimum uint64 value of 0`
   },
   REJECTED_BY_LOGIC: {
     number: 8,
-    message: "Invalid top of stack",
+    message: "Teal code rejected by logic",
     title: "Teal Rejection Error",
-    description: `Teal code was rejected because top of stack contains
-non zero value or []byte`
+    description: `Teal code rejected`
```

Comment:
We should keep the description (as why the teal code is rejected).

Reply:
yes, i think description shoud be changed because if top of stack contains non-zero value, the logic will be accepted.
will check more and update